lib/MusicBrainz/Server/Controller/Area.pm

```diff
@@ -145,6 +145,22 @@ sub releases : Chained('load')
     );
 }
 
+=head2 map
+
+Shows map of places for an area.
+
+=cut
+
+sub map : Chained('load')
```

Comment:
Good point, I'll fix it once I find the other places the name needs to be changed :)